introduction.md

```diff
@@ -5,3 +5,10 @@ Hi, my name is Carolyn Tiry. I've used git for some of my programming classes, b
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?